plugins/labhub.py

```diff
@@ -33,47 +32,51 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class LabHub(BotPlugin):
+class LabHub(DefaultConfigMixin, BotPlugin):
     """GitHub and GitLab utilities"""  # Ignore QuotesBear
 
-    GH_ORG_NAME = constants.GH_ORG_NAME
-    GL_ORG_NAME = constants.GL_ORG_NAME
+    CONFIG_TEMPLATE = {
+        'GH_ORG_NAME': None,
+        'GH_TOKEN': None,
+        'GL_ORG_NAME': None,
+        'GL_TOKEN': None,
+    }
 
-    def __init__(self, bot, name=None):
-        super().__init__(bot, name)
+    def activate(self):
+        super().activate()
 
         teams = dict()
         try:
-            gh = github3.login(token=os.environ.get('GH_TOKEN'))
+            gh = github3.login(token=self.config['GH_TOKEN'])
             assert gh is not None
-        except AssertionError:
+        except AssertionError:  # pragma: no cover, for logging
             self.log.error('Cannot create github object, please check GH_TOKEN')
         else:
-            self.GH3_ORG = gh.organization(self.GH_ORG_NAME)
+            self.GH3_ORG = gh.organization(self.config['GH_ORG_NAME'])
             for team in self.GH3_ORG.teams():
                 teams[team.name] = team
 
         self._teams = teams
 
-        self.IGH = GitHub(GitHubToken(os.environ.get('GH_TOKEN')))
-        self.IGL = GitLab(GitLabPrivateToken(os.environ.get('GL_TOKEN')))
+        self.IGH = GitHub(GitHubToken(self.config['GH_TOKEN']))
+        self.IGL = GitLab(GitLabPrivateToken(self.config['GL_TOKEN']))
 
         self.REPOS = dict()
 
         try:
             self.gh_repos = {repo.full_name.split('/')[-1]: repo for repo in
                              filter(lambda x: (x.full_name.split('/')[0] ==
-                                               self.GH_ORG_NAME),
+                                               self.config['GH_ORG_NAME']),
                                     self.IGH.write_repositories)}
-        except RuntimeError:
+        except RuntimeError:  # pragma: no cover, for logging
             self.log.exception('Something went wrong in fetching github repos.')
         else:
             self.REPOS.update(self.gh_repos)
 
         try:
             self.gl_repos = {repo.full_name.split('/')[-1]: repo for repo in
                              filter(lambda x: (x.full_name.split('/')[0] ==
-                                               self.GL_ORG_NAME),
+                                               self.config['GL_ORG_NAME']),
```

Comment:
re-create `self.GH_ORG_NAME` and `self.GL_ORG_NAME` as properties that access the config so that your patch is less noisy.